components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -152,6 +156,19 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
   }
 }
 
+function LimitReachedModal(p: { children: React.ReactNode }) {
+  return <Modal visible={true} closeable={false} onClose={()=>{}}>
+    <h3>Limit Reached</h3>
```

Comment:
**nitpick**: Minor padding issue. Same issue affects the _Running Workspaces_ modal.

```suggestion
    <h3 class="mb-4">Limit Reached</h3>
```